include/exadg/solvers_and_preconditioners/multigrid/transfers/mg_transfer_p.cpp

```diff
@@ -236,7 +236,7 @@ MGTransferP<dim, Number, VectorType, components>::do_restrict_and_add(VectorType
 
     fe_eval1.read_dof_values(src);
 
-    if(!is_dg)
+    if(not(is_dg))
```

Comment:
this one is short againa, I think you do not need the `not(...)` , but you _can_ :smile:

Reply:
*again
Italian accent all of a sudden ...